repository.yaml

```
@@ -1,3 +1,3 @@
-name: Home Assistant Addons
+name: eBUSd
```

```suggestion
name: eBUSd Add-On
```

I suggest adding 'add-on' to the name, most users should not be confused but people still mix up integrations and add-ons (and look for add-ons in HACS and stuff).